modules/ossm-extensions-smextension-format.adoc

```diff
@@ -0,0 +1,62 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: REFERENCE
+[id="ossm-extensions-smextension-format_{context}"]
+= `ServiceMeshExtension` container format
+
+You must have a `.wasm` file containing the bytecode of your WebAssembly module, and a `manifest.yaml` file in the root of the container filesystem to make your container image a valid extension image.
+
+[NOTE]
+====
+When creating new WebAssembly extensions, use WasmPlugin. ServiceMeshExtension is deprecated as of {SMProductName} version 2.2 and will be removed in a future release.
+====
+
+.manifest.yaml
+[source,yaml]
+----
+schemaVersion: 1
+
+name: <your-extension>
+description: <description>
+version: 1.0.0
+phase: PreAuthZ
+priority: 100
+module: extension.wasm
+----
+
+.Field Reference for manifest.yml
+[options="header"]
+[cols="a, a, a"]
+|===
+| Field | Description |Required
```

Comment:
For consistency (and simplicity), a suggestion that the "Required Fields" column just be "Yes" or "No"